package.json

```diff
@@ -86,5 +86,5 @@
       "test/"
     ]
   },
-  "packageManager": "pnpm@8.15.8+sha512.d1a029e1a447ad90bc96cd58b0fad486d2993d531856396f7babf2d83eb1823bb83c5a3d0fc18f675b2d10321d49eb161fece36fe8134aa5823ecd215feed392"
```

Comment:
**issue:** Even though this might seem like an improvement, this change is out of scope of this PR and also causes a complete re-rendering of pnpm-lock.yaml (due to lockfile version change)

**suggestion:** Remove this line and re-install packages using pnpm v8.15.8